Lib/ntpath.py

```diff
@@ -168,7 +168,7 @@ def splitdrive(p):
 
 
 try:
-    from nt import _path_splitroot_ex
+    from nt import _path_splitroot_ex as splitroot
```

Comment:
I suggest reformatting the fallback function's docstring to use a single-line summary:

```python
        """Split a pathname into drive, root and tail.

        The drive is defined exactly as in splitdrive(). On Windows, the root
        may be a single path separator or an empty string. The tail contains
        anything after the root. For example:

            splitroot('//server/share/') == ('//server/share', '/', '')
            splitroot('C:/Users/Barney') == ('C:', '/', 'Users/Barney')
            splitroot('C:///spam///ham') == ('C:', '/', '//spam///ham')
            splitroot('Windows/notepad') == ('', '', 'Windows/notepad')
        """
```

Use the same docstring for builtin `_path_splitroot_ex()` since it no longer has a Python wrapper function.

Reply:
I'm going to make them generic, defining a different docstring for `nt._path_splitroot_ex` & `posix._path_splitroot_ex` is not elegant.